### ISSUE TITLE
Support for skipping latest tag

### ISSUE THREAD

**Issue by bjornmagnusson** [GoogleContainerTools/jib#1427]
**Description of the issue**:
Using Gradle Plugin 1.0.0-rc2. Would like to be able to skip the **latest** tag. Reason for this is that I would like to limit building/pushing the **latest** to only our master branch builds.

**Expected behavior**:
Only build tags as specified in the tags property of to closure

**Steps to reproduce**: 
Set tags as _tags = [project.version.toString()]_

**Environment**: 
Gradle
Windows 10

**`jib-gradle-plugin` Configuration**: 
```groovy
jib {
  from {
    image = "openjdk:8"
  }
  to {
    image = "REDACTED"
    tags = ["1.0.0.SNAPSHOT"]
  }
  container {
    jvmFlags = [REDACTED]
    mainClass = mainClassName
    ports = ['8081']
    format = 'Docker'
    useCurrentTimestamp = true
  }
}

```

**Log output**: 

**Additional Information**:

**Comment by adrian-baker** [comment]
This would still be useful in the case you have no other tags to push. 

Currently it seems difficult to push a completely untagged build, which in my case I use to identify "local", throwaway builds outside of CICD (the digest is passed immediate to a cloudformation deploy script).

**Comment by mpeddada1** [comment]
@adrian-baker Please see [#3346(comment)](https://github.com/GoogleContainerTools/jib/issues/3346#issuecomment-881602362). The `:latest` is added as the default when we don't specify a tag through `to.image` or `to.tags`. This behavior is in line with how `docker` behaves.

**Comment by adrian-baker** [comment]
Yes the default behaviour is expected, it just seems hard to change if you don't want a tag at all. I'll see if I can use an extension to push an untagged image, but just wonder if it should be possible to achieve to specify this via config.

**Comment by chanseokoh** [comment]
> I'll see if I can use an extension to push an untagged image

This is not possible with an extension. The [endpoint](https://docs.docker.com/registry/spec/api/#pushing-an-image-manifest) to upload a manifest JSON should be different.

**Comment by chanseokoh** [comment]
As a hacky workaround, if you know the digest of your image that you're going to build and upload, you can specify the digest instead of a tag for `jib.to.image`. For example,
```
mvn jib:build -Djib.to.image=francium25/test@sha256:b46d0c3ce57e8424a89333d3acda3749ecbb9afb2afdcb6344882a4a9fa597f3
```
In order for this to work, the digest must match the actual digest of the image. Otherwise, upload will fail. (BTW, looks like there is no way to verify if an image is uploaded through the Docker Hub UI if it's not given any tag.) 

But of course, you should know the digest in advance. If you're build is reproducible, you can run any one of `jib:build | jib:dockerBuild | jib:buildTar` to generate `target/jib-image.digest` to get the value. (`jib:build` is preferred for maximum performance for consecutive runs.)

But I admit this is hacky.
